log/config.go

```diff
@@ -2,7 +2,11 @@ package log
 
 type typeKey int
 
-const Formatter typeKey = iota
+const (
+	Formatter typeKey = iota
+	verbosity
```

Comment:
I think it's best to separate exported and unexported symbols. Use a different type for the latter, maybe `internalTypeKey`.

Reply:
they're put in separate groups now